From 77d6bb690865540577c04952ba685a0cfe3fa01e Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Mon, 17 Jul 2017 10:53:29 +0200
Subject: [PATCH] use ifeq "$(SDK_AUTO_DEPLOYMENT)" "YES"

... instead of ifneq "$(SDK_AUTO_DEPLOYMENT)" "", as the variable is = NO
if disabled

Change-Id: I7dd2d165235f5cb7e55a8fed628ada71b04c7881
---
 odk/examples/DevelopersGuide/BasicAndDialogs/CreatingDialogs/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/odk/examples/DevelopersGuide/BasicAndDialogs/CreatingDialogs/Makefile b/odk/examples/DevelopersGuide/BasicAndDialogs/CreatingDialogs/Makefile
index bcd47ac1725f..3af249f9d4b0 100644
--- a/odk/examples/DevelopersGuide/BasicAndDialogs/CreatingDialogs/Makefile
+++ b/odk/examples/DevelopersGuide/BasicAndDialogs/CreatingDialogs/Makefile
@@ -106,7 +106,7 @@ $(COMPONENT_PACKAGE) : $(COMPONENT_JAR) $(COMPONENT_UNOPKG_MANIFEST)
 	cd $(subst /,$(PS),$(OUT_COMP_CLASS)/$(subst .$(UNOOXT_EXT),,$(@F))) && $(SDK_ZIP) -u ../../../bin/$(@F) META-INF/manifest.xml
 
 $(REGISTERFLAG) : $(COMPONENT_PACKAGE)
-ifneq "$(SDK_AUTO_DEPLOYMENT)" ""
+ifeq "$(SDK_AUTO_DEPLOYMENT)" "YES"
 	-$(DEL) $(subst \\,\,$(subst /,$(PS),$@))
 	-$(MKDIR) $(subst /,$(PS),$(@D))
 	$(DEPLOYTOOL) $(COMPONENT_PACKAGE_URL)
-- 
2.11.0

